pages/common/bats.md

```diff
@@ -0,0 +1,20 @@
+# bats
+
+> Bash Automated Testing System is a TAP (https://testanything.org/) compliant testing framework for Bash.
+> More information: <https://github.com/bats-core/bats-core>.
+
+- Run BATS tests in test script and output results in TAP (Test Anything Protocol) format:
+
+`bats --tap {{path/to/test.bats}}`
+
+- Count test cases without running any tests in the test script:
+
+`bats --count {{path/to/test.bats}}`
+
+- Run BATS tests in test scripts of a directory and subdirectories with file extension `.bats`:
```

Comment:
```suggestion
- Run tests in BATS test scripts of a directory and subdirectories with file extension `.bats`:
```

Reply:
I can't say I would be able to understand the original or the revised version of this command.

Maybe something more like:

```suggestion
- Run BATS test cases contained in a directory and subdirectories with file extension `.bats`:
```